snuba/admin/static/api_client.tsx

```diff
@@ -106,9 +106,10 @@ interface Client {
     storage_name: string,
     column_conditions: object
   ) => Promise<Response>;
+  listJobSpecs: () => Promise<JobSpecMap>;
 }
 
-function Client() {
+function Client(): Client {
```

Comment:
this enforces that we define every method in the interface, which seems good to me